pyzip-demo-db-apb/roles/provision-pyzip-demo-db-apb/tasks/main.yml

```diff
@@ -114,9 +114,11 @@
     - zipcodes.sql
 
 - name: encode bind credentials
-  shell: 'echo "{\"POSTGRES_HOST\": \"{{ dbservice_name }}\", \"POSTGRES_PORT\": \"5432\", \"POSTGRES_USER\": \"{{ database_user }}\", \"POSTGRES_PASSWORD\": \"{{ database_password }}\", \"POSTGRES_DB\": \"{{ database_name }}\"}" | base64 -w 0'
-  register: encoded_bind_credentials
+  asb_encode_binding:
```

Comment:
That would still expose how we're encoding and communicating with the broker so if we move the file to a new location other than var/tmp or use different encoding etc, people's APBs would break.  The point here was to hide that in a module that would change if the implementation in ASB changed and leave the APBs as valid.